src/android/stringsDependency.ts

```diff
@@ -28,13 +28,25 @@ export const withAndroidStringsDependency: ConfigPlugin<PluginConfigType> = (
   config,
   props
 ) => {
+  let CodePushServerURL: string = 'https://codepush.appcenter.ms/';
+
+  if (props?.android?.CodePushServerURL) {
+    CodePushServerURL = props?.android?.CodePushServerURL;
+  }
+
   if (!props?.android?.CodePushDeploymentKey) {
     throw new Error(
       "You need to provide the `CodePushDeploymentKey` Android property for the @config-plugins/react-native-code-push plugin to work."
     );
   }
 
   return withStringsXml(config, (xmlProps) => {
+    xmlProps.modResults = setStrings(
+        xmlProps.modResults,
+        "CodePushServerURL",
+        CodePushServerURL
+    );
```

Comment:
We shouldn't need to define it if it's the default, right? Let's just skip it if we can.

Something like that:

```suggestion
  if (props?.android?.CodePushServerURL) {
      xmlProps.modResults = setStrings(
          xmlProps.modResults,
          "CodePushServerURL",
          props?.android?.CodePushServerURL
      );
   }
```

Reply:
For now you need the default to point to microsoft, after App center shut down, it will be mandatory to set that up